auditbeat/module/auditd/audit_linux.go

```diff
@@ -153,6 +156,17 @@ func (ms *MetricSet) addRules(reporter mb.PushReporterV2) error {
 	}
 	defer client.Close()
 
+	// Don't attempt to change configuration if audit rules are locked (enabled == 2).
+	// Will result in EPERM.
+	status, err := client.GetStatus()
+	if err != nil {
+		return errors.Wrap(err, "failed to get audit status before adding rules")
+	}
+	if status.Enabled == auditLocked {
+		ms.log.Warn("Skipping rule configuration: Audit rules are locked")
```

Comment:
Can you push the error to the `reporter` for this case (like we do at https://github.com/elastic/beats/pull/6023/files#diff-a3559204e3ac05aabc310b7b3dfd90e3R120). As an operator I'd like to see this type of information reported in Elasticsearch.